format_checker/common_checks.py

```diff
@@ -92,7 +92,7 @@ def check_sort(filename, log):
     diff = subprocess.check_output(command, shell=True).decode("utf-8")
     if diff != "":
         log_esp_error(filename, log, "The file is not properly ordered")
-        print("Refer to IDoFT readme for how pr-data.csv should be sorted: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test")
+        print("Refer to IDoFT readme for how pr-data.csv/gr-data.csv should be sorted: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test")
```

Comment:
Change `pr-data.csv/gr-data.csv` to `pr-data.csv or gr-data.csv` (to avoid confusion with a `directory_name/file_name`). More importantly, where should README mention what `gr-data.csv` is?

Reply:
done